scripts/build/build_node/Platform/Linux/package-list.ubuntu-bionic.txt

```diff
@@ -20,6 +20,8 @@ libxkbcommon-x11-dev                    # For xcb keyboard input
 libxkbcommon-dev                        # For xcb keyboard input
 libxcb-xfixes0-dev                      # For mouse input
 libxcb-xinput-dev                       # For mouse input
+libxcb-image0-dev                       # For xcb image support
+xxd
```

Comment:
Why is `xxd` needed? It seems to be a hex dump tool

Reply:
The xxd is needed since we are doing a hex dump of PNG to the header file for the splash screen.
ssd. Please take a look here : https://github.com/o3de/o3de/pull/17913/files#diff-7c67d32f141c0b34d6aa2fd412198b83b37ed2d3645a1293862f8d84589d2cfaR32

Reply:
Ok, we'll need to load this into a new Ubuntu image, but will approve this one